applications/StructuralMechanicsApplication/custom_response_functions/adjoint_conditions/adjoint_semi_analytic_base_condition.h

```diff
@@ -178,15 +198,6 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION)  AdjointSemiAnalyticBaseCondi
 					rCurrentProcessInfo);
     }
 
-    void CalculateRightHandSide(std::vector< VectorType >& rRightHandSideVectors,
```

Comment:
What is the reason why you erased this function?

Reply:
ping @armingeiser